spring-cloud-dataflow-docs/src/main/asciidoc/howto.adoc

```diff
@@ -170,3 +170,31 @@ These properties can also be specified via `deployment` properties when deployin
 ----
 dataflow:>stream deploy foo --properties "app.*.logging.level.org.springframework.integration=DEBUG"
 ----
+
+=== Advanced SpEL expressions with stream
+
+One of the powerful feature of SpEL expressions is http://docs.spring.io/spring/docs/current/spring-framework-reference/html/expressions.html#expressions-ref-functions[functions].
+Spring Integration provides `jsonPath()` and `xpath()` out-of-the-box http://docs.spring.io/spring-integration/reference/html/spel.html#spel-functions[SpEL-functions], if appropriate libraries are in the classpath.
+All the provided Spring Cloud Stream application starters are supplied with the `json-path` and `spring-integration-xml` jars, thus we can use those SpEL-functions in Spring Cloud Data Flow streams whenever expressions are possible.
+For example we can transform JSON-aware `payload` from the HTTP request using some `jsonPath()` expression:
+
+[source,bash]
```

Comment:
Well, that is copy/paste from other similar places in this Reference Manual.
For example `streams.adoc`
Not my fault 😄